### PR TITLE
Coding site

### PR DESCRIPTION
**Pull request by Aarti002**
@harikesh409 can u review this PR , i just want to know i'm going right or not.
Although some css parts are still needs to be done. And let me know if i need to add or remove something.
Thank you!

Fixes: #479

### CODE REVIEW

**Review by harikesh409 — CHANGES_REQUESTED** [review]
- [ ] Rename the file to `suggested-coding-site.html`
- [ ] The background is very odd, don't use images just use one of the theme colours.
- [ ] Remove `coding.jpg` file which is not being used.
- [ ] All external links should be opened in a new tab.
- [ ] Navbar is not the same as the home page.
- [ ] Under "10 Best Websites to Practice Coding Online" on the hover of the links the line is extending to the hole page, correct it.
- [ ] Resolve the merge conflicts.

**Comment by Aarti002** [comment]
@harikesh409 can u add level tag? and will make require changes .
Thank you!

File: DSA.html [hunk 1]
```diff
@@ -82,6 +82,8 @@
                                 to Project</a>
 
                             <a class="dropdown-item" href="practicedsa.html">DSA Problems</a>
+                             <a class="dropdown-item" href="booklist.htmls">Suggested Books</a>
```
Comment (harikesh409) [inline DSA.html:85]: The link is wrong it should be `booklist.html`

File: suggested_coding_site.html [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+            <h2 style="color:white;">5. <a href="https://www.w3schools.com/" target="_blank">w3schools</a></h2>
+            <p class="about">
+                W3Schools has focus on simplicity. Also helps to make practice easy learning.
+W3Schools uses simple code examples and simple illustrations of how to use it.
+W3Schools' tutorials start from basic level and move all the way up to professional references.
+            </p>
+        </div>
+        <div class="card1">
+            <h3>10 Best Websites to Practice Coding Online</h3>
+            <section class="list--wrapper mt-5">
```
Comment (harikesh409) [inline suggested_coding_site.html:157]: Instead of nesting this inside, move it outside of the outer section.

File: suggested_coding_site.html [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+        <div class="card1">
+            <h2 style="color:white;">5. <a href="https://www.w3schools.com/" target="_blank">w3schools</a></h2>
+            <p class="about">
+                W3Schools has focus on simplicity. Also helps to make practice easy learning.
+W3Schools uses simple code examples and simple illustrations of how to use it.
+W3Schools' tutorials start from basic level and move all the way up to professional references.
+            </p>
+        </div>
+        <div class="card1">
+            <h3>10 Best Websites to Practice Coding Online</h3>
```
Comment (harikesh409) [inline suggested_coding_site.html:156]: Use h1 only as used above so the style is uniform.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
image
- [ ] If we click on a link in `10 Best Websites to Practice Coding Online` section and hover on another link then both tooltips are being shown and are overlapping.
- [ ] Resolve the merge conflicts.

**Comment by harikesh409** [comment]
> @harikesh409 can u add level tag? and will make require changes .
> Thank you!

While merging I will add all other labels.

**Comment by Aarti002** [comment]
@harikesh409 please review again i think everything is fine now.

**Comment by Aarti002** [comment]
@harikesh409 can you please review it as we don't have much time left and please let me know if everything is not fine?
Thank you!

**Pull request merged**